dashboard-ui/src/components/Survey/survey.jsx

```diff
@@ -406,6 +406,10 @@ class SurveyPage extends Component {
             "name": bname + ' vs ' + aname + ' vs ' + mname,
             "scenarioIndex": baselineAdm['scenarioIndex'],
             "pageType": "comparison",
+            'admAuthor': baselineAdm['admAuthor'],
+            'baselineName': baselineAdm['admName'],
+            'alignedName': alignedAdm['admnName'],
+            'misalignedName': misalignedAdm['admName'],
```

Comment:
I'm getting an error here, probably because sometimes (with the IO4 and 5 parallax adms) either aligned or misaligned don't exist. I would recommend using optional chaining here instead (alignedAdm?.admName)

Reply:
Thanks, good call